cpp/ql/src/semmle/code/cpp/exprs/Call.qll

```diff
@@ -213,7 +215,7 @@ class FunctionCall extends Call, @funbindexpr {
    * Gets the function called by this call.
    *
    * In the case of virtual function calls, the result is the most-specific function in the override tree (as
-   * determined by the compiler) such that the target at runtime will be one of result.getAnOverridingFunction*().
+   * determined by the compiler) such that the target at runtime will be one of **result.getAnOverridingFunction\*()**.
```

Comment:
Why not just use backticks here?

Reply:
I was trying to distinguish between C/C++ syntax (`monospace`) and QL syntax (**bold**).  

I stumbled upon the idea only recently and so now QL constructs are either `monospaced`, **bold**, or rendered as plain text.  What do you think this should be moving forward?  (Personally, I like the distinction between C/C++ and QL syntax, and one esp. newcomers to the platform would appreciate.)

Reply:
I think we should discuss this in a team meeting (@jbj).

Reply:
This kind of change would need to be coordinated not just between all members of our team but also across language teams. Until that has happened, please continue using our house style of monospace for all code.

Everyone is welcome to add bullets to our agenda document, so @zlaski-semmle please do that if you want to try and pitch the idea to the team.

Reply:
Ok, I've reverted to use of back-ticks for both C/C++ and QL code.  I've also added a bullet point to discuss in our C stream meetings.